<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>970</width>
    <height>560</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>970</width>
    <height>560</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>970</width>
    <height>560</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Projanager</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>-1</y>
      <width>971</width>
      <height>561</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QWidget" name="widget" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>1</horstretch>
         <verstretch>1</verstretch>
        </sizepolicy>
       </property>
       <widget class="QWidget" name="NavArea" native="true">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>10</y>
          <width>951</width>
          <height>301</height>
         </rect>
        </property>
        <widget class="QTextBrowser" name="PrefixNavigator">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>951</width>
           <height>301</height>
          </rect>
         </property>
         <property name="html">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:7.8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;TextText&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </widget>
       <widget class="Line" name="line">
        <property name="geometry">
         <rect>
          <x>-10</x>
          <y>310</y>
          <width>1001</width>
          <height>20</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QLineEdit" name="Observable">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>350</y>
          <width>351</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="WorkspaceName">
        <property name="geometry">
         <rect>
          <x>380</x>
          <y>350</y>
          <width>151</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="Prefix">
        <property name="geometry">
         <rect>
          <x>540</x>
          <y>350</y>
          <width>61</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="PrefixPath">
        <property name="geometry">
         <rect>
          <x>610</x>
          <y>350</y>
          <width>351</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="labelPathObserv">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>330</y>
          <width>251</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Путь до отслеживаемой папки</string>
        </property>
       </widget>
       <widget class="QLabel" name="labelWorkspace">
        <property name="geometry">
         <rect>
          <x>380</x>
          <y>330</y>
          <width>141</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Имя workspace</string>
        </property>
       </widget>
       <widget class="QLabel" name="labelPrefName">
        <property name="geometry">
         <rect>
          <x>540</x>
          <y>330</y>
          <width>55</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Префикс</string>
        </property>
       </widget>
       <widget class="QLabel" name="LabelPrefPath">
        <property name="geometry">
         <rect>
          <x>610</x>
          <y>330</y>
          <width>211</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Путь до папки префикса</string>
        </property>
       </widget>
       <widget class="QPushButton" name="AddingButtun">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>410</y>
          <width>941</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Добавить префикс</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="DeleteByPrefix">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>490</y>
          <width>281</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="labelDelete">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>470</y>
          <width>141</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Удаляемый префикс</string>
        </property>
       </widget>
       <widget class="QPushButton" name="DeletingButton">
        <property name="geometry">
         <rect>
          <x>310</x>
          <y>490</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Удалить</string>
        </property>
       </widget>
       <widget class="QPushButton" name="DbInit">
        <property name="geometry">
         <rect>
          <x>790</x>
          <y>460</y>
          <width>161</width>
          <height>81</height>
         </rect>
        </property>
        <property name="text">
         <string>Инициализировать 
 базу данных</string>
        </property>
       </widget>
       <widget class="Line" name="line_2">
        <property name="geometry">
         <rect>
          <x>-10</x>
          <y>440</y>
          <width>1001</width>
          <height>20</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="Line" name="line_3">
        <property name="geometry">
         <rect>
          <x>760</x>
          <y>450</y>
          <width>20</width>
          <height>121</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
       <widget class="QPushButton" name="Move">
        <property name="geometry">
         <rect>
          <x>660</x>
          <y>460</y>
          <width>81</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Перенести
файлы</string>
        </property>
       </widget>
       <widget class="Line" name="line_4">
        <property name="geometry">
         <rect>
          <x>400</x>
          <y>450</y>
          <width>20</width>
          <height>121</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
       <widget class="QPushButton" name="Observe">
        <property name="geometry">
         <rect>
          <x>430</x>
          <y>520</y>
          <width>311</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Отслеживать</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="Workspaces">
        <property name="geometry">
         <rect>
          <x>380</x>
          <y>380</y>
          <width>411</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="labelPathObserv_2">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>380</y>
          <width>141</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Путь до всех workspace</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="moveLine">
        <property name="geometry">
         <rect>
          <x>430</x>
          <y>470</y>
          <width>221</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
